db/migrate/20200519141015_create_storage_resources.rb

```diff
@@ -0,0 +1,15 @@
+class CreateStorageResources < ActiveRecord::Migration[5.1]
```

Comment:
Lets update this to 5.2 as that is the current version of rails/active_record that manageiq is using.  I assume you generated this a while ago while we were still on 5.1 but some of the defaults for e.g. references have changed so this could be simplified on the newer version